src/org/zaproxy/zap/extension/stats/StatsdClient.java

```diff
@@ -0,0 +1,269 @@
+/**
+ * StatsdClient.java
+ *
+ * (C) 2011 Meetup, Inc.
+ * Author: Andrew Gwozdziewycz <andrew@meetup.com>, @apgwoz
+ *
+ *
+ *
+ * Example usage:
+ *
+ *    StatsdClient client = new StatsdClient("statsd.example.com", 8125);
+ *    // increment by 1
+ *    client.increment("foo.bar.baz");
+ *    // increment by 10
+ *    client.increment("foo.bar.baz", 10);
+ *    // sample rate
+ *    client.increment("foo.bar.baz", 10, .1);
+ *    // increment multiple keys by 1
+ *    client.increment("foo.bar.baz", "foo.bar.boo", "foo.baz.bar");
+ *    // increment multiple keys by 10 -- yeah, it's "backwards"
+ *    client.increment(10, "foo.bar.baz", "foo.bar.boo", "foo.baz.bar");
+ *    // multiple keys with a sample rate
+ *    client.increment(10, .1, "foo.bar.baz", "foo.bar.boo", "foo.baz.bar");
+ *
+ *    // To enable multi metrics (aka more than 1 metric in a UDP packet) (disabled by default)
+ *    client.enableMultiMetrics(true);  //disable by passing in false
+ *    // To fine-tune udp packet buffer size (default=1500)
+ *    client.setBufferSize((short) 1500);
+ *    // To force flush the buffer out (good idea to add to your shutdown path)
+ *    client.flush();
+ *
+ *
+ * Note: For best results, and greater availability, you'll probably want to
+ * create a wrapper class which creates a static client and proxies to it.
+ *
+ * You know... the "Java way."
+ * 
+ * ZAP Changes:
+ * 	Originally copied from https://github.com/etsy/statsd/edit/master/examples/StatsdClient.java
```

Comment:
<https://github.com/etsy/statsd/blob/77ca6ccd3ebaa22f532fe663c51c0508c6ff4cc7/examples/StatsdClient.java> instead? (It links to exact version of the class.)
What is the license of this code? Same as statsd? Do we need to bundle the [license notice](https://github.com/etsy/statsd/blob/master/LICENSE)? Or not substantial portion?